Doc/library/time.rst

```diff
@@ -256,6 +256,13 @@ Functions
    Like :func:`gmtime` but converts to local time.  If *secs* is not provided or
    :const:`None`, the current time as returned by :func:`.time` is used.  The dst
    flag is set to ``1`` when DST applies to the given time.
+   
+   :func:`localtime` may raise :exc:`OverflowError`, if the timestamp is out of
+   the range of values supported by the platform C :c:func:`localtime` or
+   :c:func:`gmtime` functions, and :exc:`OSError` on :c:func:`localtime` or
+   :c:func:`gmtime` failure.
+   It's common for this to be restricted to years in
+   1970 through 2038. 
```

Comment:
It looks like the trailing whitespace on this line is causing the docs build to fail. 
```suggestion
   1970 through 2038.
```